### PR TITLE
Basic structure complete (minus footer) with CSS applied

### PR DESCRIPTION
**Pull request by SarahA2020**

### CODE REVIEW

File: myPortfolio.css [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+	box-sizing: border-box;
+}
+
+body {
+	padding: 0;
+	margin: 0;
+	font-size: 62.5%;
+}
+
+/* Header Section */
```
Comment (tmhnsly) [inline myPortfolio.css:11]: Good use of comments!

File: myPortfolio.css [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+	width: 60%;
+}
+
+
+.portfolioImage {
+	width: 60%;
+}
+
+/*
+.textPortfolio {
```
Comment (tmhnsly) [inline myPortfolio.css]: Couldn't see this in the HTMl, assuming it was replaced with textLayout?

File: myPortfolio.html [hunk 3]
```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<title></title>
```
Comment (tmhnsly) [inline myPortfolio.html]: No title for the tab

**Review by tmhnsly — CHANGES_REQUESTED** [review]
Looks good! I like the psuedo element in the header.

File: myPortfolio.css (hunk 2 above)
Comment (SarahA2020) [reply myPortfolio.css]: Yep, that's right. I  have deleted .textPortfolio now. Thanks

File: myPortfolio.html (hunk 3 above)
Comment (SarahA2020) [reply myPortfolio.html]: Fixed, thanks

**Review by tmhnsly — APPROVED** [review]
Looking good 😁

**Pull request merged**